launcher/ui/pages/instance/ServersPage.cpp

```diff
@@ -734,4 +792,10 @@ void ServersPage::on_actionJoin_triggered()
     APPLICATION->launch(m_inst, true, false, std::make_shared<MinecraftTarget>(MinecraftTarget::parse(address, false)));
 }
 
+void ServersPage::on_actionRefresh_triggered()
+{
+    qDebug() << "Action clicked";
+    m_model->queryServersStatus();
```

Comment:
It would be nice not to allow multiple calls.
Also consider doing this on page start also

Reply:
> Also consider doing this on page start also
Done
> It would be nice not to allow multiple calls.
If someone has a shut down server in their list, that would disable the "refresh" button until the network timeout, which may be a bad thing. In the game multiplayer screen, you can spam the "refresh" button to refresh servers even if it hasn't finished to ping every server.
So, how about keeping the current behaviour ?

Reply:
Then I would like to stop somehow the previous call:
if the user clicks multiple times the previous task should be aborted and the signals disconnected.
That way you get only the latest signals

Reply:
Please consider a mechanism to delete/stop the task that started

Reply:
Done ! ae7d3379e4ed2d8aee51026f37b7344026f97de7

Tested using `sudo tc qdisc add dev wlan0 root netem delay 5000ms` to manually slow down network, the old task's destructor is called instantly when refreshing